community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexAccessorTest.java

```diff
@@ -251,20 +251,22 @@ public void closeMustCloseAll() throws Exception
     @Test
     public void closeMustThrowIfOneThrow() throws Exception
     {
-        for ( IndexAccessor accessor : aliveAccessors )
+        for ( int i = 0; i < aliveAccessors.length; i++ )
```

Comment:
Why change from the forEach loops?

Reply:
Because the fusion instance is now stateful (closed flag) so they will all need to be recreated for for each iteration